fdbserver/Ratekeeper.actor.cpp

```diff
@@ -881,6 +927,7 @@ void Ratekeeper::updateRate(RatekeeperLimits* limits) {
 						    .detail("TargetRateRatio", targetRateRatio);
 					}
 					ssLimitReason = limitReason_t::storage_server_write_queue_size;
+					ssHighWriteQueue.push_back(ss.id);
```

Comment:
It seems the last one in `ssHighWriteQueue` is the most limiting one. Wondering if we should react on that one only, or act on all SSes involved.

Reply:
I don't know, and will think about it. Also, the ratekeeper ignores the slowest SS in a team/zone, but this PRs throttling logic does not ignore those.

Reply:
it makes sense to only react on a single SS to me.